src/dai.rs

```diff
@@ -8,7 +8,12 @@ pub const ATTOS_IN_DAI_EXP: u16 = 18;
 pub const DAI_DEC: u64 = 1_000_000_000_000_000_000;
 
 // It means the mantissa can be up to 9 digits long
-pub const FROM_ROUNDED_DAI_PRECISION: u16 = 9;
+const FROM_ROUNDED_DAI_PRECISION_EXP: u16 = 9;
+
+lazy_static::lazy_static! {
+    pub static ref FROM_ROUNDED_DAI_PRECISION: u32 =
+        10u32.pow(FROM_ROUNDED_DAI_PRECISION_EXP as u32);
+}
```

Comment:
There is a (IMO) simpler alternative :)

https://docs.rs/conquer-once/0.2.0/conquer_once/type.Lazy.html

Reply:
`A type for lazy initialization of e.g. global static variables, which provides the same functionality as the lazy_static! macro.`

Looks the same to me.

Reply:
I personally prefer it because it doesn't use macros but only regular Rust syntax, meaning I don't have to remember exactly what syntax the macro-author decided on :)